src/hooks/wallets/mpc/useMPCWallet.ts

```diff
@@ -59,42 +63,47 @@ export const useMPCWallet = () => {
 
   const tKey = useMPC()
 
+  const chain = useCurrentChain()
+
   // sets up web3
   useEffect(() => {
     const localSetup = async () => {
       const { setupWeb3 } = await import('@/hooks/wallets/mpc/utils')
 
-      if (!loginResponse) {
+      if (!loginResponse || !chain) {
         return
       }
 
       const chainConfig = {
-        chainId: '0x5',
-        rpcTarget: 'https://rpc.ankr.com/eth_goerli',
-        displayName: 'Goerli Testnet',
-        blockExplorer: 'https://goerli.etherscan.io',
-        ticker: 'ETH',
-        tickerName: 'Ethereum',
+        chainId: hexlify(Number(chain.chainId)),
```

Comment:
Is it giving anything different than `Number(chain.chainId).toString(16)`?

Reply:
Nope, I will use the your suggested conversion to reduce ethers dependencies.